editor/src/components/editor/store/editor-state.ts

```diff
@@ -191,15 +191,7 @@ export enum LeftMenuTab {
   Navigator = 'navigator',
 }
 
-export const LeftPaneMinimumWidth = 5
-
-export const LeftPaneDefaultWidth = GridMenuWidth
-export const LeftPanelMinWidth = 240
-export const DefaultNavigatorWidth = 280
-export const LeftPanelWidthAtom = atomWithPubSub({
-  key: 'LeftPanelWidthAtom',
-  defaultValue: LeftPaneDefaultWidth,
-})
+export const DefaultNavigatorWidth = 280 // todo make it GridMenuWidth
```

Comment:
Why is it not `GridMenuWidth` now?

Reply:
I'm doing that in a separate PR #4363 because it will break a bunch of tests